src/Header/SetCookie.php

```diff
@@ -298,6 +333,11 @@ public function getFieldValue()
             $fieldValue .= '; HttpOnly';
         }
 
+        $sameSite = $this->getSameSite();
+        if ($sameSite !== null && in_array($sameSite, self::SAME_SITE_ALLOWED_VALUES, true)) {
```

Comment:
$sameSite !== null must be tested because if $sameSite is null, we don't want `; SameSite=` in header and null must not be in allowed value (it breaks later)

Reply:
yeah, and as we can't set invalid value we don't need in_array check ;-)